Parse/src/main/java/com/parse/Parse.java

```diff
@@ -98,6 +101,7 @@ public Builder(Context context) {
           Context applicationContext = context.getApplicationContext();
           Bundle metaData = ManifestInfo.getApplicationMetadata(applicationContext);
           if (metaData != null) {
+            server(metaData.getString(PARSE_SERVER_URL));
```

Comment:
There is a method in line 148 that checks to make sure there is a trailing slash so I think it's OK..